containers/Containerfile

```diff
@@ -0,0 +1,91 @@
+FROM registry.fedoraproject.org/fedora-toolbox:34
+
+MAINTAINER "Candlepin Team" <candlepin@redhat.com>
+
+ENV SMDEV_CONTAINER_OFF='True'
+
+RUN yum install --disablerepo=updates -y \
+    bind-utils \
+    cmake \
+    dbus-daemon \
+    dbus-glib \
+    dbus-glib-devel \
+    desktop-file-utils \
+    dnf-utils \
+    gcc \
+    GConf2-devel \
+    gettext \
+    git \
+    git \
+    glibc-common \
+    gtk3-devel \
+    intltool \
+    json-c-devel \
+    jss \
+    langpacks-as \
+    langpacks-bn \
+    langpacks-de \
+    langpacks-es \
+    langpacks-fr \
+    langpacks-gu \
+    langpacks-hi \
+    langpacks-it \
+    langpacks-ja \
+    langpacks-kn \
+    langpacks-ko \
+    langpacks-ml \
+    langpacks-mr \
+    langpacks-or \
+    langpacks-pa \
+    langpacks-pt_BR \
+    langpacks-ru \
+    langpacks-ta \
+    langpacks-te \
+    langpacks-zh_CN \
+    langpacks-zh_TW \
+    libdnf-devel \
+    libnotify-devel \
+    make \
+    npm \
+    openssl \
+    openssl-devel \
+    postfix \
+    python3-coverage \
+    python3-dateutil \
+    python3-decorator \
+    python3-decorator \
```

Comment:
Please remove duplicity of two same lines

Reply:
Will be done on next push!